css/editCompetition.import.less

```diff
@@ -0,0 +1,26 @@
+/*
+ * Google maps input is tiny! Use styling from:
+ * https://developers.google.com/maps/documentation/javascript/examples/places-searchbox
+ */
+#competitionLocationMapInput {
+  background-color: #fff;
+  padding: 2px 11px 2px 13px;
+  width: 400px;
+  font-family: Roboto;
+  font-size: 15px;
+  font-weight: 300;
+  text-overflow: ellipsis;
+  margin-top: 20px;
+
+  &:focus {
+    border-color: #4d90fe;
+    margin-left: -1px;
+    padding-left: 14px;  /* Regular padding-left + 1. */
+    width: 401px;
```

Comment:
Haha, no idea. Looks like @jbcm627 left us a breadcrumb, though. It comes from the css on https://developers.google.com/maps/documentation/javascript/examples/places-searchbox.